framework/meta/src/cmd/standalone/template/template_adjuster.rs

```diff
@@ -20,7 +19,6 @@ pub struct TemplateAdjuster {
 impl TemplateAdjuster {
     pub fn update_dependencies(&self) {
         self.update_dependencies_root();
-        self.update_dependencies_wasm();
```

Comment:
We need a check of the template tag (framework version). This can only be removed for v0.45.0 or newer.